index.html

```diff
@@ -72,5 +88,183 @@
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+	</div>
+</section>
+<br> <br> <br>
+<section class="head">
+	<!-- Heading Section  -->
+	<div class="flex-about">
+	  <div class="about-me">
+	  	<div class="head-title"> 
+	    	About Me 
+	    </div>
+	  	<div class="head-body">
+	  		Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+	  	</div>
+	  	<div class="see-3">
+	  		Get My Resume
+	  	</div>
+	  	<div>
+	  		<br><br><br><br><br><br><br>
+	  		<hr>
+	  	</div>
+	  </div> 
+	  <div>
+	  	<br>
+	  	<br>
+	  	<span class="lang-title"> <img src="assets/images/rectangle.png" alt="rec"> Language </span> 
+	  	<div class="lang">
+		  	<p>JavaScript</p>
+		  	<p>Ruby</p>
+		  	<p>HTML</p>
+		  	<p>CSS</p>
+	  	</div>
+	  </div>
+	  <div>
+	  	<span class="lang-title"> <img src="assets/images/rectangle.png" alt="rectangle"> Frameworks </span> 
+	  	<div class="lang">
+		  	<p>Ruby On Rail</p>
+		  	<p>RSPec</p>
+		  	<p>Copy Bara</p>
+		  	<p>Selenium</p>
+	  	</div>
+	  </div>
+	  <div>
+	  	<span class="lang-title"> <img src="assets/images/rectangle.png" alt="rectangle"> Skills </span> 
+	  	<div class="lang">
+		  	<p>CodeKit</p>
+		  	<p>GitHub</p>
+		  	<p>Code Pen</p>
+		  	<p>GitLab</p>
+		  	<p>Terminal</p>
+	    </div>
```

Comment:
- [ ] Kindly be noticed that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.
- [ ] And the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.